### ISSUE TITLE
Cannot parse ExAC VCF

### ISSUE THREAD

**Issue by LeeTL1220** [broadinstitute/oncotator#343]
pyVCF

```
SyntaxError: One of the INFO lines is malformed: ##INFO=<ID=DP_HIST,Number=R,Type=String,Description="Histogram for DP; Mids: 2.5|7.5|12.5|17.5|22.5|27.5|32.5|37.5|42.5|47.5|52.5|57.5|62.5|67.5|72.5|77.5|82.5|87.5|92.5|97.5">
```

**Comment by LeeTL1220** [comment]
Solved with latest (unreleased master branch as of this writing) version of PyVCF.

https://github.com/jamescasbon/PyVCF

**Comment by LeeTL1220** [comment]
- [ ] Update build venv script
- [ ] Unit tests all pass
- [ ] Update ``Dockerfile``
- [ ] Update pysam version to support ExAC downloaded index.

**Comment by LeeTL1220** [comment]
Closed by #344
